Source/Engine/Core/Types/Variant.h

```diff
@@ -39,6 +39,9 @@ API_STRUCT(InBuild) struct FLAXENGINE_API VariantType
         Vector2,
         Vector3,
         Vector4,
+        Int2,
```

Comment:
Move new enum entries to the end - otherwise, it will break serialization (those in-built types are serialized as `byte`).